.travis.yml

```diff
@@ -1,7 +1,19 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
-  - "0.12"
-  - "0.11"
   - "0.10"
+  - "0.11"
+  - "0.12"
+  - "iojs"
+  - 4
+  - 6
+  - 8
+  - 10
+  - 12
+  - node
```

Comment:
Let's not include this one. I'd prefer a reproducible CI.

Reply:
You got it! 5bb137a493b7ff7abf2fb871aad530ada0ce3b9b